autoload/slime.vim

```diff
@@ -25,7 +25,8 @@ function! s:ScreenSend(config, text)
         \ " -X eval \"readreg p " . g:slime_paste_file . "\"")
   call system("screen -S " . shellescape(a:config["sessionname"]) . " -p " . shellescape(a:config["windowname"]) .
         \ " -X paste p")
-  call system('screen -X colon "
"')
+  call system('screen -X colon "
+"')
```

Comment:
is that a merge/reformat include?

let's revert it?

Reply:
Ahh, yeah. Had the same problem for the initial commit, but then I noticed it. Forgot about it for this small fix.